packages/client-api-schema/package.json

```diff
@@ -31,15 +17,32 @@
   "lint-staged": {
     "index.ts": "yarn lint:check --max-warnings=0"
   },
-  "main": "types/index.d.ts",
-  "repository": "statechannels/monorepo/blob/master/packages/client-api-schema",
   "scripts": {
-    "build": "ts-node index.ts",
-    "build:ci": "ts-node index.ts",
-    "lint:check": "eslint index.ts",
-    "lint:write": "eslint index.ts --fix",
-    "precommit": "lint-staged --quiet",
+    "build": "yarn build:typescript",
+    "build:generate-types": "ts-node build.ts",
+    "build:typescript": "npx tsc -b",
+    "build:ci": "ts-node build.ts",
+    "lint:check": "eslint \"src/**/*.ts\" --cache",
+    "lint:write": "eslint \"src/**/*.ts\" --fix",
     "prepare": "yarn build"
   },
-  "types": "types/index.d.ts"
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/statechannels/monorepo/blob/master/packages/client-api-schema"
+  },
+  "author": "snario <liam@l4v.io>",
+  "contributors": [
+    "Liam Horne <liam@l4v.io>",
+    "Alex Gap <alex.gap@consensys.net>"
+  ],
+  "devDependencies": {
+    "@types/eslint": "^6.1.3",
+    "@types/eslint-plugin-prettier": "^2.2.0",
+    "@typescript-eslint/eslint-plugin": "^2.10.0",
+    "@typescript-eslint/parser": "^2.10.0",
+    "eslint": "^6.7.2",
+    "eslint-plugin-prettier": "^3.1.1",
+    "json-schema-to-typescript": "^8.0.0",
+    "lint-staged": "9.4.0"
+  }
```

Comment:
Was it necessary to change these dependencies? Could this be the source of your linting woes?

Reply:
Hmm. This happened during a rebase. Will try and fix.